src/Common/Select.php

```diff
@@ -307,6 +307,20 @@ public function removeCol($alias)
 
     /**
      *
+     * Has the column or alias been added to the query?
+     *
+     * @param string $alias The column or alias to look for
+     *
+     * @return bool
+     *
+     */
+    public function hasCol($alias)
+    {
+        return isset($this->cols[$alias]) || array_search($alias, $this->cols) !== false;
```

Comment:
see https://github.com/auraphp/Aura.SqlQuery/pull/112/files/9fb886d8955663988cdfb5c371416ccdc7cdbe27#diff-1e1715bbfd6b71bb485e70bb72002475R99. cols() can add a column or an alias. This finds both, similar to how removeCol() works.